src/main/java/fiftyfive/administration/usermanagement/dto/CreateUserRequest.java

```diff
@@ -0,0 +1,39 @@
+package fiftyfive.administration.usermanagement.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+
+import java.time.LocalDateTime;
+
+@Data
+public class CreateUserRequest {
+    private String username;
+    private String password;
+    @JsonProperty("first_name")
+    private String firstName;
+    @JsonProperty("last_name")
+    private String lastName;
+    private String role;
+    @JsonProperty("created_at")
+    private LocalDateTime createdAt=LocalDateTime.now();
+    @JsonProperty("updated_at")
+    private LocalDateTime updatedAt=LocalDateTime.now();
```

Comment:
Can remove these field from request body since we will not be sending the date in request